xbmc/music/GUIViewStateMusic.cpp

```diff
@@ -188,6 +189,8 @@ CGUIViewStateMusicDatabase::CGUIViewStateMusicDatabase(const CFileItemList& item
       AddSortMethod(SortByArtist, sortAttribute, 557, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));  // Filename, empty | Userdefined, Userdefined
       // year
       AddSortMethod(SortByYear, 562, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));
+      // album date added
+      AddSortMethod(SortByDateAdded, sortAttribute, 570, LABEL_MASKS("%F", "", strAlbumLeft + " - " + strAlbumRight, "%a"));  // Filename, empty | Userdefined - Userdefined, dateAdded
```

Comment:
TBH I have no clue why album listings are handled this special but I'd prefer to keep the current format with this addition and then discuss any visual improvements/changes in a separate PR because otherwise this whole addition is going to be blocked over such a minor point.